index.html

```diff
@@ -47,6 +47,45 @@ <h2>Filtres</h2>
           <!--add icon-->Plus de 500 logements sont disponibles dans cette ville
         </p>
       </section>
+      <section id="most-popular">
+        <h3>Les plus populaires</h3>
+        <!--add icon-->
+        <div>
+          <a href="#">
+            <img
+              src="/Users/stephanemalho/Documents/MalhoStephane_2_27012021/assets/images/hebergements/4_small/emile-guillemot-Bj_rcSC5XfE-unsplash.jpg"
```

Comment:
Path must be relative "./assets/image/etc" so add folder image inside a folder assets on project